### ISSUE TITLE
Compatibility with Ansible 2.13.x

### ISSUE THREAD

**Issue by binaryfire** [mitogen-hq/mitogen#966]
Hi guys

Looks like Ansible 2.13 was added to the Ubuntu PPA last week. I just ran apt-get upgrade and I'm now getting the following error:

```
ERROR! Your Ansible version ((2, 13, 4)) is too recent. The most recent version
supported by Mitogen for Ansible is (2, 12).x. Please check the Mitogen
release notes to see if a new version is available, otherwise
subscribe to the corresponding GitHub issue to be notified when
support becomes available.
```

Would it be possible to add 2.13 compatibility?

Cheers (and thanks for your hard work on this project)

**Comment by arrfab** [comment]
same for CentOS Stream 8 and 9 so it will be landing in RHEL 8 and RHEL 9 soon : 
```
rpm -q ansible-core
ansible-core-2.13.3-1.el8.x86_64
```
Reverting to 2.12 for the time being (and blocking updates for that pkg)

**Comment by opoplawski** [comment]
Look like support was added to master with #933

**Comment by opoplawski** [comment]
Although with current master I hit issue #952

**Comment by guenhter** [comment]
I guess this one can be closed because Ansible 2.13 support is now supported by https://github.com/mitogen-hq/mitogen/commit/e8ad12e881a04e7b49048645ec88a3afbc304f14

**Comment by moreati** [comment]
duplicate of  #929
